Update §3.2 Applying Preferences and §4 Vocabulary Definition (#213)
ietf-wg-aipref/drafts: draft-ietf-aipref-vocab.md

## Applying Preferences {#applicability}

This specification enables the expression of a defined set of preferences that
can be communicated and interoperably understood. Readers of this
specification should understand that it does not:

> - provide for enforcement of these preferences;

Reviewer: I think the original language was slightly better here:
"not provide a means of ensuring that preferences are respected"

Reviewer 2: Thanks Mirja,

I'll let @TimidRobot decide how to deal with this, but I'm assuming that this is a friendly amendment.

Reviewer 2: I prefer the "followed" option, or the even more terse, but perhaps better still:

```suggestion
- ensure that preferences are followed;
```

Reviewer: I don't think "respected" is normative but followed or MT's last suggest is fine. I really just think enforcement is too strong/a trigger for people assuming there should be an enforcement mechanism. We could even say "applied"; that would be even more neutral (not implying that there is a general expectation that all preferences would always be followed by "well-behaved" entities).

- address if, how, or when preferences should be followed or ignored;
- address external mechanisms -- technical, legal, contractual, or otherwise --

>   that might determine adherence or non-adherence to these preferences;

Reviewer 2: Is "determine" the right word here?  At some level, we don't want to imply that adherence to regulation is discretionary, but this really will come down to setting up incentives rather than imperatives.

Put differently, part of the point of this section is to emphasize that respecting preferences is -- to some extent -- discretionary.  That it might not be a choice at some level is the point of this statement, but the implication from using "determine" is that there could be a straight switch.

How about:

> 
```suggestion
- address technical, legal, contractual, or other mechanisms
  that might create a stronger requirement to follow preferences;
```

- consider situations or purposes that might justify following or not-following
  expressed preferences.

Because of this, stakeholders need to decide when and how to follow or
not-follow preferences in the context of legal, institutional, ethical, or

> other interests and commitments.

Reviewer 2: ~~This is my iteration on the suggestion from Sebastian on list.~~  I'll add his too.

(See below)

Reviewer: External to what? Maybe s/external/different/

Reviewer 2: There is also the possibility that "whether to follow" is better as "whether to follow or not follow"

Reviewer 2: External to the interactions between the entity with this document.  In this case, any ambiguity is fine.

Reviewer 2: On the "external" point.

```suggestion
An entity that receives usage preferences has a choice
whether to follow those preferences.
Multiple factors could influence that decision,
but the decision itself is outside the scope of this document.
```

After consideration, though I do think that centering the focus on the document would be a good thing (to Sebastian's point during discussion), this sentence puts the document at the pointy end of the paragraph (the end) rather than the other point end (the start) and that's fine.

Reviewer 2: I think that I prefer the "Multiple factors" variant here.  Other than that, this is close enough.  Thanks for asking.

> other interests and commitments.

Reviewer 2: Sebastian's text, which has received considerable support on the list:

```suggestion
An entity that receives usage preferences has a choice
whether to follow those preferences.
This specification does not determine how that choice is made.
Whether and under which circumstances a preference is followed
is outside the scope of this specification.
```

Reviewer 3: Sorry for jumping in late - how about a pure procedural description and scoping? 

```suggestion
An entity that receives usage preferences must 
determine how to apply those preferences.
This specification does not define how that 
determination is made.
```

Reviewer: I  like that last proposal; it puts it more neutral.

## Applying Preferences {#applicability}

This specification enables the expression of a defined set of preferences that
can be communicated and interoperably understood. Readers of this
specification should understand that it does not:

- ensure that preferences are followed;
- address if, how, or when preferences should be followed or not-followed;
- address technical, legal, contractual, or other mechanisms that might create
  a stronger requirement to follow or not follow preferences;
- consider situations or purposes that might justify following or not-following
  expressed preferences.

> An entity that receives usage preferences has a choice whether to follow those

Reviewer 2: I'm just going to leave this here: https://www.prindleinstitute.org/2025/04/why-follow-the-law/
